altimeter/core/resource/resource_spec.py

```diff
@@ -192,6 +192,8 @@ def merge_resources(resource_id: str, resources: List[Resource]) -> Resource:
             resources = spec_classes_resources[winning_class]
         full_type_names = {resource.type for resource in resources}
         merged_resource_type_name = full_type_names.pop()
+        account_ids = [resource.account_id for resource in resources]
+        merged_account_id = account_ids.pop()
```

Comment:
Why are you doing this? If the resources have different account_id shouldn't you raise an Exception? It seems to me that they should not be merged.